components/TokenBalance/TokenDeposit.tsx

```diff
@@ -325,7 +315,7 @@ export const TokenDeposit = ({
           </div>
 
           <div className="flex flex-col mt-6 space-y-4 sm:flex-row sm:space-x-4 sm:space-y-0">
-            {hasTokensInWallet || inAccountDetails ? (
+            {/* {hasTokensInWallet || inAccountDetails ? (
```

Comment:
explain why? this may make sense but just checking

Reply:
yeah I just looked over this and it will surely break things?

Reply:
I know the component hierarchy here is annoying, but this component is used in many places, and hiding the deposit button in all of them isnt going to be right.